composeCore/src/main/java/com/hamdan/forzenbook/compose/core/theme/Dimens.kt

```diff
@@ -137,3 +149,10 @@ object IconSizeValues {
     val giga_1 = 200.dp
     val giga_2 = 220.dp
 }
+
+object StrokeValues {
+    val stroke1 = 2.dp
```

Comment:
Why do you need a million strokes?

Reply:
because I need many strokes, changed to be just 2, 2.dp for small 4.dp for large